### PR TITLE
Update CI and dependencies

### PR DESCRIPTION
**Pull request by p-galligan**
Update CI settings in travis file.

Includes dependency updates.

### CODE REVIEW

**Comment by p-galligan** [comment]
@bonniegee or @helrond I wouldn't mind some insight into why this branch errored. The message is less than helpful. Any ides?

**Comment by p-galligan** [comment]
Is it possibly this line in the travis file? 

`- cp ${APPLICATION_NAME}/config.py.example ${APPLICATION_NAME}/config.py`

**Comment by helrond** [comment]
So it looks like the build shows as failed because the deployment failed. Have you checked the error message in AWS CodeDeploy?

**Comment by p-galligan** [comment]
Thanks for the tip! Figured it out based on the logs.

**Comment by helrond** [comment]
@p-galligan IMO better visibility on deployments is another thing to throw into the deployment strategy thinking...

File: requirements.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django==3.2.4
+Django==3.2.5
```
Comment (helrond) [inline requirements.in]: I think most recent Django is 3.2.7?

**Review by helrond — APPROVED** [review]

**Pull request merged**